### ISSUE TITLE
Feature Request: NPM Package management support

### ISSUE THREAD

**Issue by arizvisa** [cpitclaudel/z3.wasm#1]
Adding an index.js that exports some of your funcs from demo.js and a package.json will expose this to the whole javascript community. This'll get you a whole ton more users and thus better support anyways.

(You could probably use viz-js from https://github.com/mdaines/viz.js as a framework, as it's essentially the same thing but for building a different target).

**Comment by cpitclaudel** [comment]
> Adding an index.js that exports some of your funcs from demo.js and a package.json will expose this to the whole javascript community

Indeed, that would be great. I just haven't had time (and proper experience with JS packaging would help, too.

Help welcome :)

**Comment by arizvisa** [comment]
Since you're using node.js which typically comes with npm, I'm pretty sure all you really need to do is run npm init.

Just keep in mind that you're exposing your project to a bunch of js devers :-p, so if you never wanted a career in maintaining software, then good luck with that. ;-)

**Comment by sim642** [comment]
@arizvisa I've created a similar project but I've published it on npm also: https://www.npmjs.com/package/z3em.
